src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/Example.rst

```diff
@@ -0,0 +1,252 @@
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_PorePressureBC -->
+  :end-before: <!-- SPHINX_PorePressureBCEnd -->
+
+-----------------------------------------------------------
+Collecting output data
+-----------------------------------------------------------
+
+It is convenient to collect data in hdf5 format that can be easily post-processed using Python. To collect the temperature field for all the time steps, the following XML blocks need to be defined:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_TemperatureCollection -->
+  :end-before: <!-- SPHINX_TemperatureCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_TemperatureTimeHistory -->
+  :end-before: <!-- SPHINX_TemperatureTimeHistoryEnd -->
+
+Similarly, the following blocks are needed to collect the solid stress:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressCollection -->
+  :end-before: <!-- SPHINX_SolidStressCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SolidStressTimeHistory -->
+  :end-before: <!-- SPHINX_SolidStressTimeHistoryEnd -->
+
+The displacement field can be collected using ``nodeManager`` as follows
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementCollection -->
+  :end-before: <!-- SPHINX_DisplacementCollectionEnd -->
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementTimeHistory -->
+  :end-before: <!-- SPHINX_DisplacementTimeHistoryEnd -->
+
+Also, periodic events are required to trigger the collection of this data on the mesh. For example, the periodic events for collecting the displacement field are defined as:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/THM_wellbore_benchmark.xml
+  :language: xml
+  :start-after: <!-- SPHINX_DisplacementPeriodicEvent -->
+  :end-before: <!-- SPHINX_DisplacementPeriodicEventEnd -->
+
+
+---------------------------------
+Results and benchmark
+---------------------------------
+
+A good agreement between the GEOSX results and analytical results for temperature and pore pressure distribution around the wellbore is shown in the figures below:
+
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/THM_wellbore_temperature_pressure.py
+
+.. _problemThermoPoroElasticWellbore_Temperature_Pressure_Fig:
+.. figure:: temperature_pressure.png
+   :align: center
+   :width: 800
+   :figclass: align-center
+
```

Comment:
Please remove this redundant figure, which should be generated by python script.

Reply:
The plots require hdf5 data files that are too large such that I cannot push them to github. Therefore they must be done in local to create the figures that are actually displaced in the doc. The script are provided for local use only.